app/views/admin/loans/questionnaires/breakeven_rampup/_rampup_table.html.slim

```diff
@@ -0,0 +1,26 @@
+tr.group-content
+  td
+    td.text-center colspan=periods
+      = units
```

Comment:
I don't think it's valid to nest a td in a td like this. Was that intentional? Otherwise, I think the `br` method is fine.